app/blueprints/fund_builder/templates/round.html

```diff
@@ -20,9 +20,10 @@ <h1 class="govuk-heading-l">{{ pageHeading }}</h1>
                             "id": form.fund_id.id,
                             "name": form.fund_id.name,
                             "label": {
-                            "text": form.fund_id.label
+                            "text": "Select Fund"
```

Comment:
Better to keep the older one if this is the agreed name use this

Reply:
Done please review